RPIPs/RPIP-46.md

```diff
@@ -1,108 +0,0 @@
----
-rpip: 46
-title: Universal Adjustable Revenue Split
-description: Allow the revenue from borrowed ETH (aka, rETH commission) to be split between fixed targets; the relative split between targets is adjustable.
-author: Valdorff (@Valdorff)
-discussions-to: TBD
-status: Draft
-type: Protocol
-category: Core
-created: 2024-03-08
-requires: potentially rpip-45 or rpip-50
-tags: tokenomics-2024, tokenomics-content
----
-
-## Abstract
-Currently, the commission determines the payout of revenue split between rETH and each specific minipool. Other parties, such as RPL, gain value indirectly. This proposal allows for splitting revenue between four initial parties: reth (the main product), node operators (the decentralized operators actually staking), voters (a subset of operators that have vote power), and a surplus revenue mechanism.
-
-This proposal also includes a small set of items for potential future use:
-- Seals that the security council can use to increase the NO share -- this is being used to find a reasonable setting based on the actual market.
-- An allowlist of controllers that may make changes to the settings, which allows for potential automation in the future
-
-## Specification
-1. Inflation settings SHALL be modified to retain inflation to the DAOs and eliminate inflation to NOs
-   1. `rpl.inflation.interval.rate` SHALL be set to `1000040763630249500` (1.5% per year)
-   2. Node Operators (`rocketClaimNode`) allocation SHALL be set to 0%
-   3. pDAO (`rocketClaimDAO`) allocation SHALL be set to 95%
-   4. oDAO (`rocketClaimTrustedNode`) allocation SHALL be set to 5%
-2. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `voter_share`, `surplus_share`
```

Comment:
Are these pDAO settings? Security Council permission over them? etc. this one is fairly long, try to separate into headings, pDAO settings is maybe a good heading.

Reply:
They are settings. pDAO and oDAO can both set settings. Security council only when explicitly allowed to control a specific setting.